WorkloadManagementSystem/JobWrapper/JobWrapperTemplate.py

```diff
@@ -140,19 +140,19 @@ def execute( arguments ):
       gLogger.error( 'Failed to execute job', result['Message'] )
       raise JobWrapperError( (result['Message'], result['Errno']) )
   except JobWrapperError as exc:
-    if exc[1] == 0 or str(exc[0]) == '0':
+    if exc.value[1] == 0 or str(exc.value[0]) == '0':
```

Comment:
I don't manage to imagine what 'value' could look like. A comment ? Or how about directly parsing value upon initialization of JobWrapperError ?

Reply:
Defined here: https://github.com/DIRACGrid/DIRAC/blob/integration/WorkloadManagementSystem/JobWrapper/JobWrapperTemplate.py#L40

Reply:
This I saw, but what I do not manage to imagine is an iterable whose either first element is a int as a string, or its second element a proper int.